src/routes/components/components.json

```
@@ -2932,5 +2932,18 @@
 		"category": "Forms & User Input",
 		"tags": ["inputs and widgets", "forms", "interactions"],
 		"stars": 0
-	}
+	},
+        {
+                "title": "Lucide icons",
```

The file need to have tabulations, not spaces.
```suggestion
	{
		"title": "Lucide icons",
```
You can run `npm run format` to fix it with prettier

---

Oh thanks, didn't noticed!